cla_public/templates/accessibility-statement.html

```diff
@@ -90,28 +103,50 @@ <h2 class="govuk-heading-m">{{ _('Content that’s not within the scope of the a
     {% trans %}Not applicable{% endtrans %}
   </p>
 
-  <h2 class="govuk-heading-m">{{ _('How we tested this service') }}</h2>
+  <h2 class="govuk-heading-m">{{ _('What we’re doing to improve accessibility') }}</h2>
 
   <p class="govuk-body">
-    {% trans %}This service was last tested on 22 July 2020. The test was carried out by the Digital Accessibility Centre (DAC).{% endtrans %}
```

Comment:
Do we want to keep this in the translation file ?

Reply:
Yes